domains/buffer.json

```
@@ -0,0 +1,16 @@
+{
+  "description": "bufferwise",
+  "repo": "https://github.com/bufferwise/bufferwise.github.io",
+  "owner": {
+    "username": "bufferwise",
+    "email": "bufferwise@gmail.com",
+    "Discord": "790846560229392444",
+    "Discord Server": "https://discord.gg/26MMXRHgZB",
+    "Instagram": "bufferwise",
+    "Twitter / X": "bufferwise"
```

Discord server is not needed
```suggestion
    "username": "bufferwise",
    "email": "bufferwise@gmail.com",
    "discord": "790846560229392444",
    "instagram": "bufferwise",
    "twitter": "bufferwise"
```